stock_picking_package_preparation_line/views/stock_picking_package_preparation_line.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="0">
+
+    <record id="sppp_line_stock_picking_package_preparation_form" model="ir.ui.view">
+        <field name="name">sppp.line.stock.picking.package.preparation.form</field>
+        <field name="model">stock.picking.package.preparation</field>
+        <field name="inherit_id" ref="stock_picking_package_preparation.stock_picking_package_preparation_form"/>
+        <field name="arch" type="xml">
+            <page string="Transfers" position="after">
+                <page string="Details">
+                    <field name="line_ids">
+                        <tree>
+                            <field name="sequence" widget="handle"/>
+                            <field name="name" />
+                            <field name="product_id"/>
+                            <field name="product_uom_qty" />
+                            <field name="product_uom" groups="product.group_uom"/>
+                        </tree>
+                        <form>
+                            <group>
+                                <field name="product_id"
+                                       attrs="{'readonly': [('move_id', '!=', False)]}"/>
+                                <field name="product_uom_qty"
+                                       attrs="{'required': [('product_id', '!=', False)]}"/>
+                                <field name="product_uom" groups="product.group_uom"
+                                       attrs="{'required': [('product_id', '!=', False)], 'readonly': [('move_id', '!=', False)]}"/>
+                            </group>
+                            <field name="name" colspan="4"
+                                   nolabel="1" placeholder="Description"/>
+                            <field name="move_id" invisible="1"/>
+                            <field name="note" colspan="4"
+                                   nolabel="1" placeholder="Note"/>
```

Comment:
hmmm... not sure it is the better place. In the following capture (with note) the difference between the description and the note is not really visible, what do you think ?

![image](https://cloud.githubusercontent.com/assets/7602419/9821413/72cdf250-58ba-11e5-9b7d-269f334f3b38.png)